store/mem/store.go

```diff
@@ -49,4 +49,5 @@ func (s Store) CacheWrapWithTrace(w io.Writer, tc types.TraceContext) types.Cach
 func (s *Store) Commit() (id types.CommitID) { return }
 
 func (s *Store) SetPruning(pruning types.PruningOptions) {}
+func (s *Store) GetPruning() types.PruningOptions        { return types.PruningOptions{} }
```

Comment:
why do we return an empty pruning option and set pruning is a no-op? adding comments would be helpful for future maintainers

Reply:
I don't follow? These types just implement the `Committer` interface. However, they're not needed here -- comments are redundant.

Reply:
what I'm trying to say is that a comment about why they are not needed would be helpful

Reply:
done :)